reana_server/cli.py

```diff
@@ -20,11 +20,11 @@
 generated using the ``secrets`` python library.
 
 To retrieve the administrator token you can use the
-`reana-cluster <https://github.com/reanahub/reana-cluster>`_ CLI:
+`reana-dev <https://github.com/reanahub/reana>`_ CLI:
```

Comment:
Since `reana-dev` is supposed to be known to all developers, and since the target link may seem cryptic as to where to find it, perhaps simply remove the link... as people studying `reana-server` should have come across `reana-dev` already.